app/decorators/miq_ae_method_decorator.rb

```diff
@@ -1,5 +1,14 @@
 class MiqAeMethodDecorator < MiqDecorator
-  def self.fonticon
-    'ff ff-method'
+  def fonticon
+    case location
+    when "playbook"
+      'fa fa-font'
+    when "inline"
+      'fa fa-diamond'
```

Comment:
@epwinchell This diamond is for ruby method so we might need it to be colored red.

Reply:
@epwinchell let's create a new icon class called `fa-diamond-red`, what do you think?